src/core_atmosphere/Registry.xml

```diff
@@ -255,6 +255,31 @@
                      units="-"
                      description="Maximum w-damping coefficient at model top"
                      possible_values="0 $\leq$ config_xnutr $\leq$ 1"/>
+
+                <nml_option name="config_horiz_absorbing_layer" type="character" default_value="none" in_defaults="false"
+                     units="-"
+                     description="h_diffusion configuration for upper absorbing layer"
+                     possible_values="`none' or `mpas_cam'"/>
+
+                <nml_option name="config_mpas_cam_coef" type="real" default_value="0.2" in_defaults="false"
+                     units="-"
+                     description="coefficient for scaling the 2nd-order h-mixing in mpas_cam absorbing layer"
+                     possible_values="0 $\leq$ config_mpas_cam_coef $\leq$ 1"/>
+
+                <nml_option name="config_rayleigh_damp_u" type="logical" default_value="false" in_defaults="false"
```

Comment:
Rather than having a separate logical to activate the Rayleigh damping on u, we could just apply the damping whenever the specified timescale is greater than zero.

Reply:
As per our discussion, a smaller coefficient increases damping, so we'll leave the logical for now.